plugins/nodes/src/js/components/NodesTable-new.tsx

```diff
@@ -1,64 +1,77 @@
 import * as React from "react";
-import { Table, Column } from "@dcos/ui-kit";
+import { Table, Column } from "@dcos/ui-kit/dist/packages";
 
 import NodesList from "#SRC/js/structs/NodesList";
 import Node from "#SRC/js/structs/Node";
 
 import { SortableColumnHeader } from "ui-kit-stage/SortableColumnHeader";
-import { SortDirection } from "../types/SortDirection";
+import { SortDirection } from "#PLUGINS/nodes/src/js/types/SortDirection";
```

Comment:
Was this your import plugin? IMHO relative paths are better to reference things within a plugin.

Reply:
Uh, I think so 🤔 cant remember changing them. shall I revert?

Reply:
Would love to 😇